src/convertToObject.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
+// const complexStylesString = require('./stylesString');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newString = sourceString.split(';');
+
+  const objects = newString
+    .map((element) => element.split(':'))
+    .filter((item) => item.length > 1);
+
+  const result = {};
+
+  objects
+    .map((item) => [item[0].trim(), item[1].trim()])
+    .forEach((item) => {
+      return item[0].includes('-')
+        ? (result[`${item[0]}`] = item[1])
+        : (result[item[0]] = item[1]);
```

Comment:
The conditional operation inside `forEach` is not necessary. Since you are using the property name as a key in the result object, you can directly assign the value without checking if it includes a '-'.